external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/test/KafkaSpoutTopologyMainNamedTopics.java

```diff
@@ -94,10 +98,10 @@ protected StormTopology getTopologyKafkaSpout() {
 
     protected KafkaSpoutConfig<String,String> getKafkaSpoutConfig() {
         ByTopicRecordTranslator<String, String> trans = new ByTopicRecordTranslator<>(
-                (r) -> new Values(r.topic(), r.partition(), r.offset(), r.key(), r.value()),
```

Comment:
I find it surprising that the lambda function does not work here.  I remember explicitly checking to see if it would produce a static lambda or not, and I thought it would create a static one.